Docs/configuration user view (#162)
CombiRWTH/StaffScheduling: docs/user-view/configuration/index.md

> Welcome to the configuration overview of our project! This document aims to provide users with a clear understanding of the various settings available within our application.

Reviewer: ```suggestion
This chapter aims to give users a clear understanding of the various configuration options available in our application.
```

> Our application features two modes: the full version, which connects to a centralized database, and the light version, designed for use without database access. In the light version without database access you can adjust all configurations, in contrast to the full version
> where many configurations are already defined through the database.

Reviewer: ```suggestion
The application has two modes: a fully integrated version that connects to a centralized database and a light version designed for use without database access.
With the light version, you can adjust all configurations.
In contrast, many configurations are already defined through the database in the full version.
```

> In this overview, we will differentiate between user-adjustable configurations that can be modified at any time and those specific to the light version.

Reviewer: ```suggestion
This overview will explain the difference between configurations that can be modified at any time by the user and configurations specific to the light version.
```

Let’s explore the available configurations!

## User-Adjustable Configurations

- **Forbidden Days and Shifts**: Configure specific days or shifts when certain employees are not allowed to work [here](/user-view/configuration/forbidden-days).

> - **Rounds Permission**: Set up which employees need to be available for early weekday rounds (*german: Visiten*) [here](/user-view/configuration/rounds-permission). Our application ensures at least one of these employees is assigned an early shift.

Reviewer: ```suggestion
- **Rounds Permission**: Set up which employees need to be available for early weekday rounds (*german: Visiten*) [here](/user-view/configuration/rounds-permission). Our application ensures at least one of these employees is assigned to an early shift.
```

> - **Minimum Number of Staff**: Define the minimal number of staff required per type ("Hilfskraft", "Fachkraft" or "Azubi") for each day of the week and shift [here](/user-view/configuration/minimal-staff).

Reviewer: All the links must be updated to (./minimal-staff.md) e.g

Reviewer: No this doesn't work in the deployed version, see https://combirwth.github.io/StaffScheduling/ all the links are referencing the root

Author: Done. How about the links used for the snippets. Will they work? In mkdocs.yml the base path is defined as "docs"

- **Qualification Mapping**: Assign each qualification label from the database to one of the three types: "Hilfskraft", "Fachkraft" or "Azubi" [here](/user-view/configuration/qualifications).

- **Weights**: Adjust the importance of objectives in the general objective function [here](/user-view/configuration/weights).

> - **Blocked Shifts and Days:** Adjust availability of employees manually (!= vacation days) [here](/user-view/configuration/blocked-shifts), e.g. block all night shifts, if person is not allowed to work at night or block each thursday if employee does not have to work on thursdays.

Reviewer: ```suggestion
- **Blocked Shifts and Days:** Adjust availability of employees manually (!= vacation days) [here](/user-view/configuration/blocked-shifts), e.g. block all night shifts if the person is not allowed to work at night, or block each Thursday if the employee does not work on Thursdays.
```
